modules/storage/openstack.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2022 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+const (
+	// OpenStack is the global ServiceType that refers to all the components deployed
+	// by the meta-operator
+	OpenStack ServiceType = "Openstack"
+)
+
+// OpenStackExtraVolMounts exposes the main parameters processed by the underlying
+// operators and defines the common VolMounts structure provided by the storage module
+type OpenStackExtraVolMounts struct {
+	// +kubebuilder:validation:Optional
+	Name string `json:"name,omitempty"`
+	// +kubebuilder:validation:Optional
+	Region string `json:"region,omitempty"`
```

Comment:
Yes, they're eventually needed to perform additional filtering on top of the basic propagation we provide. I'm really not sure these parameters are needed at this stage (because we're not using them), but I see:

1. potential work that can be done when at some point openstackcontrolplane will support multiple deployments
2. it's an example that shows how an operator related data structure can be extended